client/lib/plans/features-list.js

```diff
@@ -1229,11 +1236,14 @@ export const FEATURES_LIST = {
 		getSlug: () => constants.FEATURE_PRODUCT_SCAN_REALTIME_V2,
 		getIcon: () => ( { icon: 'security', component: MaterialIcon } ),
 		getTitle: () =>
+			( {
+				i5: i18n.translate( 'Real-time Scan (automated)' ),
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **19** times:
`translate( 'Automated real-time scanning' )` *ES Score: 10*